src/QueryEditor.tsx

```diff
@@ -105,28 +105,21 @@ export class QueryEditor extends PureComponent<Props, QueryEditorState> {
             onChange={this.onQueryChange}
             onRunQuery={this.props.onRunQuery}
           />
-        </div>
-        <div className="gf-form">
-          <Collapse
-            collapsible
-            label="Options"
-            isOpen={this.state.isOptionsOpen}
-            onToggle={() => this.setState({ isOptionsOpen: !this.state.isOptionsOpen })}
-          >
-            <Field
-              label="Legend"
-              description="Series name override or template. Ex. {{hostname}} will be replaced with label value for hostname."
-            >
-              <Input
-                css
-                name="legendFormat"
-                spellCheck="false"
-                onChange={this.onChangeFormat}
-                value={this.props.query.format}
-              />
-            </Field>
-          </Collapse>
-        </div>
+        </InlineField>
+        <InlineField
+          grow
+          label="Series name"
+          labelWidth={15}
+          tooltip="Series name displayed in the legend. Interpolation of variables supported with 'Variable syntax', eg $varname."
```

Comment:
Do we still need the option to substitute dashboard variables into series legend? I hardly can imagine a case when it will be necessary, do you? Like series represent dimensions, but variables stays for filtering within dimensions afaik. 
And if that there's no point to keep format field if we can use refId.